### DISCUSSION TITLE
What happens when multiple juicefs processes are mounted but share different caches?

### DISCUSSION THREAD

**Discussion by vipsland** [juicedata/juicefs#2662] [Q&A]
What happens when multiple juicefs processes are mounted but share different caches?

if i mount juicefs #1 with tikv using cache1
if i mount juicefs #2 with tikv using cache2 

1. get on juicefs#2 (cache2 cached)
1. deletion on juicefs #1 (cache1 deleted)
2. get on juicefs#2 (cache2... gotten?) <- will this happen?

if so, will there be a mechanism to stop this from happening in future?

**Comment by davies** [comment]
Assume you are mount the same volume with two different process. What's does the `deletion on juicefs 1` mean?
